package/upgrade/lib.sh

```diff
@@ -222,14 +222,14 @@ upgrade_addon()
   local namespace=$2
   local version=$3
 
-  version=$(cat ./addons/${name}.yaml | yq e .spec.version)
+  version=$(cat /usr/local/share/addons/${name}.yaml | yq e .spec.version)
```

Comment:
seems   `version=$(yq e '.spec.version' /usr/local/share/addons/${name}.yaml)` is straightforward